app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
Actually, you're right. And we could follow this for instance? https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/